seeder/seeder.js

```diff
@@ -4,66 +4,73 @@
   const writer = csvWriter();
+  for (var j = 1; j < 10; j++) {
+    writer.pipe(
+      fs.createWriteStream('./CSV/product/product.csv' + j, { flags: 'a' })
+    )
 
-  writer.pipe(
-    fs.createWriteStream('./CSV/product.csv', { flags: 'a' })
-  )
-
-  for (let i = 1; i <= 9000000; i++) {
-    const rating = helper.randomRating();
-    const dataOne = {
-      product_id: i,
-      product_name: 'Placeholder',
-      comfort_average: rating,
-      durability_average: rating,
-      size_average: rating,
-      stars_average: rating
-    }
-    if (!writer.write(dataOne)) {
-      await new Promise((resolve) => writer.once('drain', resolve));
-    }
-  };
-  writer.end()
-  return 'success';
+    for (let i = 1; i <= 1000000; i++) {
+      const dataOne = {
+        id: i,
+        product_id: i,
+        product_name: 'Placeholder',
+        comfort_average: helper.randomRating(),
+        durability_average: helper.randomRating(),
+        size_average: helper.randomRating(),
+        stars_average: helper.randomRating()
+      }
+      if (!writer.write(dataOne)) {
+        await new Promise((resolve) => writer.once('drain', resolve));
+      }
+    };
+  }
+  return 'done';
 }
 
 const genReviews = async (names, dates) => {
   const writer = csvWriter();
+  let totalProd = 0;
+  for (var i = 1; i < 11; i++) {
+    writer.pipe(
+      fs.createWriteStream('./CSV/review/review.csv' + i, { flags: 'a' })
+    )
 
-  writer.pipe(
-    fs.createWriteStream('./CSV/review.csv', { flags: 'a' })
-  )
-
-  for (let j = 1; j <= 1000000; j++) {
-    var rand = helper.randomRating();
-    const dataTwo = {
-      product_id: Math.floor(Math.random() * (9000000 - 1) + 1),
-      username: helper.randomName(names),
-      comment: lorem.generateSentences(2),
-      verified: helper.randomBool(),
-      locale: 'US',
-      upvote: rand,
-      downvote: rand,
-      created_at: helper.randomDate(dates),
-      size_rating: rand,
-      durability_rating: rand,
-      comfort_rating: rand,
-      stars: rand,
-    };
-    if (!writer.write(dataTwo)) {
-      await new Promise((resolve) => writer.once('drain', resolve));
+    for (let j = 1; j <= 100000; j++) {
+      var currentProdID = totalProd += 1;
+      for (let k = 0; k < 5; k++) {
+        const dataTwo = {
+          id: j,
+          product_id: currentProdID,
+          username: helper.randomName(names),
+          comment: lorem.generateSentences(2),
+          verified: helper.randomBool(),
+          locale: 'US',
+          upvote: helper.randomRating(),
+          downvote: helper.randomRating(),
+          created_at: helper.randomDate(dates),
+          size_rating: helper.randomRating(),
+          durability_rating: helper.randomRating(),
+          comfort_rating: helper.randomRating(),
+          stars: helper.randomRating()
+        };
+        if (!writer.write(dataTwo)) {
+          await new Promise((resolve) => writer.once('drain', resolve));
+        }
+      }
```

Comment:
This all makes sense.
Pulling the total records to start off where the last left off.
Using the async/ await to ensure write is done.